packages/flutter_tools/pubspec.yaml

```diff
@@ -20,7 +20,7 @@ dependencies:
   crypto: 3.0.6
   ffi: 2.1.3
   file: 7.0.1
-  flutter_template_images: 4.2.0
+  flutter_template_images: 4.3.0
```

Comment:
I believe you'll have to run `flutter update-packages --force-upgrade` to recompute this file's checksum in line 124. I'd run it with the `--cherry-pick-package/version` option to only bump this package and not everything else.

Reply:
Looks like you can't mix --cherry-pick _and_ --force-upgrade, but just --chery-pick-xxx worked.